docs/do-not-merge.md

```diff
@@ -0,0 +1,377 @@
+
+# Highlights
+
+<!-- HIGHLIGHTS H2. FOR EACH MAJOR FEATURE FOR THE RELEASE -->
+
+## Query from deep storage (experimental)
+
+Druid now supports querying segments that are stored only in deep storage. When you query from deep storage, you increase the data available for queries without necessarily having to scale your Historical processes to accommodate more data. To take advantage of the potential savings, make sure you configure your load rules to not load all your data onto Historical processes. Then, configure a corresponding drop rule to drop those segments.
+
+Note that at least one segment of a datasource must be loaded onto a Historical process so that Druid can plan the query. It can be any segment though.
+
+As part of this feature, there are new API endpoints available to interact with the query:
+
+- Submit queries using `POST /sql/statements`. The payload is the same as any other Druid SQL query with additional context parameter available. 
+  - The required `executionMode: ASYNC` configures Druid to return the results asynchronously. Results are only available after the query completes.
+  - The optional context parameter `selectDestination:DURABLE_STORAGE` configures Druid to write the query results to durable storage.
+
+- Get the status of a query with `GET /sq/statements/:queryID`.
+
+- Get query results with `/sql/statements/:queryID`
+
+[14416](https://github.com/apache/druid/pull/14416)
+[14512](https://github.com/apache/druid/pull/14512)
+[14527](https://github.com/apache/druid/pull/14527)
+
+## Java 17 support
+
+Druid now fully supports Java 17.
+[14384](https://github.com/apache/druid/pull/14384)
+
+## Array column types
+
+Array column types are now generally available.
+
+For more information about this feature, see the [26.0.0 release notes for array column types](https://github.com/apache/druid/releases#26.0.0-highlights-auto-type-column-schema-%28experimental%29).
+
+## Schema auto-discovery
+
+Schema auto-discovery is now generally available.
+
+For more information about this feature, see the [26.0.0 release notes for Schema auto-discovery](https://github.com/apache/druid/releases#26.0.0-highlights-auto-type-column-schema-%28experimental%29-schema-auto-discovery-%28experimental%29).
+
+## Smart segment loading
+
+Revamped Coordinator to make it more stable and user-friendly. This is accompanied by several bug fixes, logging and metric improvements, and a whole new range of capabilities.
+
+Coordinator now supports a `smartSegmentLoading` mode, which is enabled by default. When enabled, users don't need to specify any of the following dynamic configs as they would be ignored by the coordinator. Instead, the coordinator computes the optimal values of the following configs at run time to best utilize coordinator runs:
+
+* `maxSegmentsInNodeLoadingQueue`
+* `maxSegmentsToMove`
+* `replicationThrottleLimit`
+* `useRoundRobinSegmentAssignment`
+* `useBatchedSegmentSampler`
+* `emitBalancingStats`
```

Comment:
```suggestion
* `replicantLifetime`
* `maxNonPrimaryReplicantsToLoad`
* `decommissioningMaxPercentOfMaxSegmentsToMove`
```

The two configs `emitBalancingStats` and `useBatchedSegmentSampler` have been completely removed as indicated in the "Cluster managmeent" section below. 

 `replicantLifetime`, `maxNonPrimaryReplicantsToLoad` and `decommissiongMaxPercent` are also included under `smartSegmentLoading` now.